components/GalleryPictures.tsx

```diff
@@ -6,7 +6,7 @@ type Props = {};
 
 function GalleryPictures({}: Props) {
```

Comment:
so is this component supposed to be a carousel?  it's confusing that you have a `Gallery` which has several `GalleryPictures` (and "pictures" is plural)

Reply:
Try and re-explain this because this didn't make much sense to me.

Reply:
"Gallery" already implies a collection of images , so it's confusing to call something Gallery which in turn contains multiple GalleryPictures

GalleryCarousel for instance would be a more descriptive name (if accurate)

Reply:
Well I was thinking "This is the Gallery" and then "This is the pictures to use FOR the gallery"